### ISSUE TITLE
Show calender state in cache details for events

### ISSUE THREAD

**Issue by Bananeweizen** [cgeo/cgeo#6910]
Now that we have the calendar functionality completely inside the app, shouldn't it be possible to also query the calendar and show an additional ", In calendar" at the state line on the details tab for every event that is actually in the calendar (and not expired yet)? So the state line could then be for example "Will attend, In calendar". I would like that myself very much since I have added caches multiple times in the past...

**Comment by Lineflyer** [comment]
AFAIR from the change we made to get the calendar functionality into c:geo itself, this is now no longer possible. We do not request the permission to access the calendar but now use a standardized intent to send the event to the calendar app. IMHO this is a one way?!

**Comment by pstorch** [comment]
Yes, it's only one way.

**Comment by Bananeweizen** [comment]
Well, then this is only possible after #6905 has been done.

**Comment by Lineflyer** [comment]
Yes, with on-demand permissions we could easily request calendar permission again without needed to explain in general, why we do so.
This would also allow to incorporate the contacts plugin function into the main app.
